code/modules/uplink/uplink_items.dm

```diff
@@ -712,7 +712,7 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	item = /obj/item/storage/box/syndie_kit/chameleon
 	cost = 4
```

Comment:
There is one problem. A revolver costs 4, A Esword costs 8. This costs 4. This means in low pop you can have the perfect disguise and esword and revolver people to death.
Maybe we need a cant buy it together system for lowpop?

Reply:
Disguises don't really work on lowpop, you can just buy carp and freely murderbone once you killed the one robust person on the server

Reply:
why not just limit buying easy instant kill items on lowpop instead of trying to fix an item that barely works on lowpop